server/service/src/sync/translations/vvm_status.rs

```diff
@@ -0,0 +1,95 @@
+use repository::{vvm_status_row::{VVMStatusRow, VVMStatusRowDelete}, StorageConnection, SyncBufferRow};
+
+use serde::Deserialize;
+
+use super::{PullTranslateResult, SyncTranslation};
+
+#[allow(non_snake_case)]
+#[derive(Deserialize)]
+pub struct LegacyVVMStatusRow {
+    ID: String,
+    description: String,
+    code: String,
+    level: i32,
+    is_active: bool,
+    unusable: bool,
```

Comment:
```suggestion
    #[serde(default)]
    unusable: bool,
```

This will mean if the value is missing it will default to false. I think it's good idea to do this in case an older version of mSupply is used we want to make sure we don't fail to sync the record.